css/properties/background-image.json

```diff
@@ -169,7 +169,14 @@
                 },
                 {
                   "prefix": "-webkit-",
-                  "version_added": "21"
+                  "version_added": "113"
+                },
+                {
+                  "prefix": "-webkit-",
+                  "version_added": "20",
+                  "version_removed": "113",
+                  "partial_implementation": true,
+                  "notes": "Support for <code>url</code> images only and <code>x</code> is the only supported resolution unit."
```

Comment:
I nearly suggested rewriting this note wholesale, but then I saw that you're copying an existing one. Letting it stand for consistency. 😄

Reply:
Feel free to rewrite them all in a separate PR :)